sdk/servicebus/Azure.Messaging.ServiceBus/src/Diagnostics/ServiceBusEventSource.cs

```diff
@@ -992,12 +1023,12 @@ public virtual void CreateReceiveLinkStart(string identifier)
             }
         }
 
-        [Event(CreateReceiveLinkCompleteEvent, Level = EventLevel.Informational, Message = "Receive link created for Identifier: {0}.")]
-        public virtual void CreateReceiveLinkComplete(string identifier)
+        [Event(CreateReceiveLinkCompleteEvent, Level = EventLevel.Informational, Message = "Receive link created for Identifier: {0}. Session Id: {1}")]
+        public virtual void CreateReceiveLinkComplete(string identifier, string sessionId)
         {
             if (IsEnabled())
             {
-                WriteEvent(CreateReceiveLinkCompleteEvent, identifier);
+                WriteEvent(CreateReceiveLinkCompleteEvent, identifier, sessionId);
```

Comment:
Do we need to guard against null here for `sessionId`?

Reply:
I don't think so.. passing null sessionId seems to be fine.